doc/dev/health-reports.rst

```diff
@@ -0,0 +1,126 @@
+==============
+Health Reports
+==============
+
+
+How to Get Reports
+==================
+
+In general, there are two channels to retrieve the health reports:
+
+ceph (CLI)
+   which send ``health`` mon command for retrieving the health status of the cluster
+mgr module
+   which calls ``mgr.get('health')`` for the same report in the form of a JSON encoded string
+
+Following diagrams outline the involved parties and how the interact when the clients
```

Comment:
s/Following/The following/, I guess (but I'm not a native speaker).

Reply:
fixed.